pkg/sql/delegate/show_tables.go

```diff
@@ -68,7 +68,10 @@ FROM %[1]s.pg_catalog.pg_class AS pc
 LEFT JOIN %[1]s.pg_catalog.pg_roles AS rl on (pc.relowner = rl.oid)
 JOIN %[1]s.pg_catalog.pg_namespace AS ns ON (ns.oid = pc.relnamespace)
 %[4]s
-LEFT JOIN crdb_internal.table_row_statistics AS s ON (s.table_id = pc.oid::INT8)
+LEFT JOIN
+(
+  SELECT DISTINCT ON (table_id) table_id, estimated_row_count FROM crdb_internal.table_row_statistics
```

Comment:
out of curiosity, do we achieve the same thing by adding a `crdb_internal.table_row_statistics(table_id)` virtual index? i forget if virtual indexes are required to be unique

Reply:
I don't think they are unique indexes.